renderers/server-render.jsx

```diff
@@ -0,0 +1,172 @@
+import Rx from 'rxjs';
```

Comment:
Okay, sorry. Some of this stuff is getting a little confusing w/ the refactoring + two repos.  I did some digging to make sure I actually understand this.

- I *think* one confusion for me was that in the mup-web app we build `build/server-locales/<locale>/serverRender.js` as the app's server entry which I assumed was `src/server-render.jsx`, but I see now is `src/server.js`.

- Another source of confusion:

> the server entry point(s) are in the individual apps, which have src/server.js or src/server-locale as the entry point for the locale bundles and scripts/start.js or whatever for the app server bundle.

I didn't even realize `build/server-locale/` was a thing now! 

- My takeaway:

I think it just comes down to the names of the source code files and the build output files in the application code at this point, since `build/server.js` corresponds to `scripts/start.js` and `build/server-locale/<locale>/serverRender.js` corresponds to `scripts/server.js`. Could we have corresponding output names?

- instead of `scripts/start.js` have `scripts/server.js` => `build/server.js`
- instead of `scripts/server.js` have `scripts/serverRender.js` => `build/server-locale/.../serverRender.js`

Or something similar.